expandFns.php

```diff
@@ -92,24 +92,33 @@ function wikify_external_text($title) {
   }
   $title = preg_replace('~[\*]$~', '', $title);
   $title = title_capitalization($title, TRUE);
-  
-  // The following two do not allow < within the inner match since the end tag is the same :-( and they might nest or who knows what
-  $title = preg_replace_callback('~(?:<Emphasis Type="Italic">)([^<]+)(?:</Emphasis>)~iu',
-      function ($matches) {return ("<i>" . $matches[1]. "</i>");},
-      $title);
-  $title = preg_replace_callback('~(?:<Emphasis Type="Bold">)([^<]+)(?:</Emphasis>)~iu',
-      function ($matches) {return ("<b>" . $matches[1]. "</b>");},
-      $title);
-  
-  $originalTags = array("<i>","</i>", '<title>', '</title>',"From the Cover: ");
-  $wikiTags = array("''","''",'','',"");
+
   $htmlBraces  = array("&lt;", "&gt;");
   $angleBraces = array("<", ">");
-  $title = sanitize_string(// order of functions here IS important!
-             str_ireplace($originalTags, $wikiTags, 
-               str_ireplace($htmlBraces, $angleBraces, $title)
-             )
-           );
+  $title = str_ireplace($htmlBraces, $angleBraces, $title);
+
+  $originalTags = array('<title>', '</title>', 'From the Cover: ');
+  $wikiTags = array('','','');
+  $title = str_ireplace($originalTags, $wikiTags, $title);
+
+  $title_orig = '';
+  while ($title != $title_orig) {
+    $title_orig = $title;  // Might have to do more than once.   The following do not allow < within the inner match since the end tag is the same :-( and they might nest or who knows what
+    $title = preg_replace_callback('~(?:<Emphasis Type="Italic">)([^<]+)(?:</Emphasis>)~iu',
+      function ($matches) {return ("'" . $matches[1] . "'");},
```

Comment:
Two ''s for italics?
```suggestion
      function ($matches) {return ("''" . $matches[1] . "''");},
```